improver_tests/acceptance/test_phase_change_level.py

```diff
@@ -41,54 +41,27 @@
 run_cli = acc.run_cli(CLI)
 
 
-def test_snow_sleet(tmp_path):
-    """Test snow/sleet level calculation"""
+@pytest.mark.parametrize("phase_type", ("snow-sleet", "sleet-rain",
+                         "sleet-rain-unfilled"))
+def test_phase_change(tmp_path, phase_type):
+    """Test phase change level calculation"""
     kgo_dir = acc.kgo_root() / f"{CLI}/basic"
-    kgo_path = kgo_dir / "snow_sleet_kgo.nc"
+    kgo_name = "{}_kgo.nc".format(phase_type.replace("-", "_"))
+    kgo_path = kgo_dir / kgo_name
     output_path = tmp_path / "output.nc"
     input_paths = [kgo_dir / x for x in
                    ("wet_bulb_temperature.nc",
                     "wbti.nc",
                     "orog.nc",
                     "land_mask.nc")]
     args = [*input_paths,
-            "--phase-change", "snow-sleet",
-            "--output", output_path]
-    run_cli(args)
-    acc.compare(output_path, kgo_path)
-
-
-def test_sleet_rain(tmp_path):
-    """Test sleet/rain level calculation"""
-    kgo_dir = acc.kgo_root() / f"{CLI}/basic"
-    kgo_path = kgo_dir / "sleet_rain_kgo.nc"
-    output_path = tmp_path / "output.nc"
-    input_paths = [kgo_dir / x for x in
-                   ("wet_bulb_temperature.nc",
-                    "wbti.nc",
-                    "orog.nc",
-                    "land_mask.nc")]
-    args = [*input_paths,
-            "--phase-change", "sleet-rain",
-            "--output", output_path]
-    run_cli(args)
-    acc.compare(output_path, kgo_path)
-
-
-def test_sleet_rain_unfilled(tmp_path):
-    """Test sleet/rain level calculation leaving below orography points
-    unfilled."""
-    kgo_dir = acc.kgo_root() / f"{CLI}/basic"
-    kgo_path = kgo_dir / "sleet_rain_unfilled_kgo.nc"
-    output_path = tmp_path / "output.nc"
-    input_paths = [kgo_dir / x for x in
-                   ("wet_bulb_temperature.nc",
-                    "wbti.nc",
-                    "orog.nc",
-                    "land_mask.nc")]
-    args = [*input_paths,
-            "--phase-change", "sleet-rain",
-            "--horizontal-interpolation", "False",
-            "--output", output_path]
+            "--phase-change"]
+    horiz_interp = "True"
+    if len(phase_type.split("-")) > 2:
```

Comment:
The problem here is what if we add another test.
Pytest.parameterize does take a list of tuples for the second argument
```
@pytest.mark.parametrize("phase_type", [("snow-sleet", False), ("sleet-rain", False), ("sleet-rain-unfilled", True)])
```

Reply:
Good shout! I'll update it now